src/main/domain/io.fusionauth.domain.search.BaseSearchCriteria.json

```diff
@@ -3,9 +3,6 @@
   "type" : "BaseSearchCriteria",
   "description" : "/**\n * @author Brian Pontarelli\n */\n",
   "fields" : {
-    "maxHitCount" : {
```

Comment:
I believe this is from Brent's `maxHitCount` changes that were rolled back and had not been regenerated yet.